### ISSUE TITLE
Helm creation of postgreql fails when installing multiple AWX on different namespaces on same kubernetes

### ISSUE THREAD

**Issue by themr0c** [ansible/awx#2991]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME

 - Installer

##### SUMMARY

The name of the postgresql installed by helm is defined by `kubernetes_deployment_name`, which is not necessarly changed when you change the namespace.

```
kubernetes_deployment_name: awx
```
As a result, if AWX is already deployed on a first namespace, deployment on a second namespace will fail at this step:

```
- name: Deploy and Activate Postgres (Kubernetes)
  shell: |
    helm install --name {{ kubernetes_deployment_name }} --namespace {{ kubernetes_namespace }} \
      --set postgresqlUsername={{ pg_username }} \
      --set postgresqlPassword={{ pg_password }} \
      --set postgresqlDatabase={{ pg_database }} \
      --set persistence.size={{ pg_volume_capacity|default('5')}}Gi \
      --tiller-namespace={{ tiller_namespace | default('kube-system') }} \
      stable/postgresql
  when:
    - pg_hostname is not defined or pg_hostname == ''
    - postgres_svc_details is defined and postgres_svc_details.rc != 0
    - kubernetes_context is defined
  register: kubernetes_pg_activate
  no_log: yes
```

##### ENVIRONMENT
* AWX version: 2.1.2
* AWX install method: kubernetes
* Ansible version:  2.7.5

##### STEPS TO REPRODUCE

- deploy awx on a first namespace
- deploy awx on a second namespace

##### EXPECTED RESULTS

successful AWX deployment

##### ACTUAL RESULTS

```
TASK [kubernetes : Deploy and Activate Postgres (Kubernetes)] ***************************************************
Thursday 10 January 2019  16:53:58 +0100 (0:00:00.167)       0:01:21.303 ****** 
Thursday 10 January 2019  16:53:58 +0100 (0:00:00.167)       0:01:21.299 ****** 
fatal: [awx]: FAILED! => changed=true 
  censored: 'the output has been hidden due to the fact that ''no_log: true'' was specified for this result'
```

##### ADDITIONAL INFORMATION

I don't know what would make more sense:

- Use `kubernetes_namespace` instead of `kubernetes_deployment_name` for:

```
 helm install --name {{ kubernetes_namespace }} --namespace {{ kubernetes_namespace }} 
```

Or 

- Redefine kubernetes_deployment_name: "{{ kubernetes_namespace }}"

**Comment by matburt** [comment]
This is a good report, not such a high priority for us so I'll mark it as help-wanted. If you get a chance to submit a PR we'll review it.

**Comment by themr0c** [comment]
Very fine for me.

I had some afterthoughts with the 2 solutions that I proposed. I see unwanted side effects in both options.

The most generic solution that i would propose is following:

```
helm install --name {{ kubernetes_namespace }}-{{ kubernetes_deployment_name }} --namespace {{ kubernetes_namespace }}
```

If you agree, I will propose a PR with that change.

For our installation, without changing the playbook, I  finally came to following workaround:  give `kubernetes_deployment_name` and `kubernetes_namespace` same value, and as the value is calculated from the `dockerhub_version`, we have no overlap given our usecase.

```
kubernetes_deployment_name: "awx-{{ dockerhub_version | replace('.', '-') }}"
kubernetes_namespace: "awx-{{ dockerhub_version | replace('.', '-') }}"
```

**Comment by themr0c** [comment]
fixed Merge pull request #2999
